src/containerapp/azext_containerapp/custom.py

```diff
@@ -2366,7 +2366,7 @@ def enable_dapr(cmd, name, resource_group_name,
     if 'configuration' not in containerapp_def['properties']:
         containerapp_def['properties']['configuration'] = {}
 
-    if 'dapr' not in containerapp_def['properties']['configuration']:
+    if not containerapp_def['properties']['configuration'].get('dapr', None):
```

Comment:
consider using safe_get() to handle such null object in response.

Reply:
Thank you. Will use that instead 👍